src/nanotron/config/config.py

```diff
@@ -189,7 +185,7 @@ class ModelArgs:
     """Arguments related to model architecture"""
 
     model_config: NanotronConfigs
-    init_method: Union[RandomInit, ExistingCheckpointInit]
+    init_method: Union[RandomInit, SpectralMupInit, ExistingCheckpointInit]
```

Comment:
Should mu transfer be external to nanotron or a nanotron core feature ?

Reply:
It should be a core feature of nanotron imo, because people can just set init_method to mup, then nanotron do it automatically under the hood...